grpc-health-probe.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: grpc-health-probe
+  version: 0.4.18
+  epoch: 0
+  description: A command-line tool to perform health-checks for gRPC applications in Kubernetes and elsewhere
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/grpc-ecosystem/grpc-health-probe
+      tag: v${{package.version}}
+      expected-commit: 89e1a1152440ebaad28bdfd0b17a01500b16ca7f
+
+  - runs: |
+      mkdir -p ${{targets.destdir}}/usr/bin
+      CGO_ENABLED=0 go build -a -tags netgo -ldflags=-w -trimpath -o ${{targets.destdir}}/usr/bin/grpc_health_probe .
```

Comment:
ah, we can _almost_ do this with https://edu.chainguard.dev/open-source/melange/melange-pipelines/go/build/!